packages/test-runner-commands/src/sendKeysPlugin.ts

```diff
@@ -0,0 +1,41 @@
+import { TestRunnerPlugin } from '@web/test-runner-core';
+import type { ChromeLauncher } from '@web/test-runner-chrome';
+import type { PlaywrightLauncher } from '@web/test-runner-playwright';
+
+type SendKeysPayload = { type?: string; press?: string };
+
+export function sendKeysPlugin(): TestRunnerPlugin<SendKeysPayload> {
+  return {
+    name: 'send-keys-command',
+    async executeCommand({ command, payload = {}, session }): Promise<any> {
+      if (command === 'send-keys') {
+        // handle specific behavior for playwright
+        if (session.browser.type === 'playwright') {
+          const page = (session.browser as PlaywrightLauncher).getPage(session.id);
+          if (payload.type) {
```

Comment:
warn on unknown options or including both `type` and `press`?

Reply:
Makes sense. Would you rather thing in the warnings or in the type that you asked for in `commands.mjs`? If we went `{keys:string} | {press:string}` for a type, we could manage this is TS primarily?

Reply:
Added.